platform/android/docs/camera/animation-types.md

```diff
@@ -1,13 +1,66 @@
 # Animation Types
 
+!!! note
+
+    You can find the full source code of this example in [`CameraAnimationTypeActivity.kt`](https://github.com/maplibre/maplibre-native/blob/main/platform/android/MapLibreAndroidTestApp/src/main/java/org/maplibre/android/testapp/activity/activity/camera/CameraAnimationTypeActivity.kt) of the MapLibreAndroidTestApp.
+
+## Animation Types
```

Comment:
Nit: This is the same heading title as the one above

Reply:
Good point!